dev.py

```diff
@@ -46,6 +46,29 @@ def cmd_all_npm_build(args):
         frontend_dir = next(project_dir.glob("*/frontend/"))
         run_verbose(["npm", "run", "build"], cwd=str(frontend_dir))
 
+
+def cmd_all_install_python_deps(args):
+    """"Install all dependencies needed to run e2e tests for all examples and templates"""
+    for project_dir in EXAMPLE_DIRECTORIES + TEMPLATE_DIRECTORIES:
+        run_verbose(["pip", "install", "-e", ".[devel]"], cwd=str(project_dir))
+
+
+def cmd_all_install_browsers(args):
+    """"Install multiple browsers to run e2e for all examples and templates"""
+    for project_dir in EXAMPLE_DIRECTORIES + TEMPLATE_DIRECTORIES:
+        e2e_dir = next(project_dir.glob("**/e2e/"), None)
+        if e2e_dir:
+            run_verbose(["playwright", "install", "webkit", "chromium", "firefox", "--with-deps"], cwd=str(project_dir))
+
+
+def cmd_all_run_e2e(args):
+    """"Run e2e tests for all examples and templates"""
+    for project_dir in EXAMPLE_DIRECTORIES + TEMPLATE_DIRECTORIES:
```

Comment:
We have two components that have the same package name, so we need to make sure that the tests are run against the correct package.
https://github.com/streamlit/component-template/tree/master/template-reactless/my_component
https://github.com/streamlit/component-template/tree/master/template

Reply:
True, they have the same package name, but different paths.
I find the e2e directory just to make sure that the component already has e2e tests. 
If yes, then I run `pytest` command in project dirs like: `/template-reactless` and `/template`. 
So no way for incorrect package.
Or maybe I am missing something?

Reply:
We test packages that are installed on the system, not those that are in a particular directory, but I checked and if we set the CWD to the path of the package, everything works.
```
$ python -c 'import my_component; print(my_component.__file__)' [CWD: examples/RadioButton]
/Users/kbregula/Snowflake/streamlit/component-template/template-reactless/my_component/__init__.py
$ python -c 'import my_component; print(my_component.__file__)' [CWD: examples/CustomDataframe]
/Users/kbregula/Snowflake/streamlit/component-template/template-reactless/my_component/__init__.py
$ python -c 'import my_component; print(my_component.__file__)' [CWD: examples/SelectableDataTable]
/Users/kbregula/Snowflake/streamlit/component-template/template-reactless/my_component/__init__.py
$ python -c 'import my_component; print(my_component.__file__)' [CWD: examples/MaterialLogin]
/Users/kbregula/Snowflake/streamlit/component-template/template-reactless/my_component/__init__.py
$ python -c 'import my_component; print(my_component.__file__)' [CWD: template]
/Users/kbregula/Snowflake/streamlit/component-template/template/my_component/__init__.py
$ python -c 'import my_component; print(my_component.__file__)' [CWD: template-reactless]
/Users/kbregula/Snowflake/streamlit/component-template/template-reactless/my_component/__init__.py
```

Reply:
I see. I checked it also and it works indeed 👍